force-app/main/default/classes/SOQL.cls

```diff
@@ -162,48 +265,94 @@ public with sharing class SOQL {
      * @return The total revenue.
      */
     public static Decimal getTotalRevenueByStage(String stage) {
+        // Initialize a variable to store the total revenue
+        Decimal totalRevenue;
+        
         // Use AggregateQuery to get the total revenue
-
+        AggregateResult[] results = [
+            SELECT SUM(Amount) totalAmount 
+            FROM Opportunity 
+            WHERE (StageName = :stage 
+            OR Amount > 10000)
+            ];
+        
+        // Retrieve the total amount from the AggregateResult
+        totalRevenue = (Decimal)results[0].get('totalAmount');
+        
         // Return the total revenue
-        return null; // Replace null with the variable you used to store the result
-    }
+        return totalRevenue;
+    }       //PASS
 
     /**
      * Question 13
      * Retrieves the first 10 Accounts ordered by Name.
      * @return A list of the first 10 Accounts ordered by Name.
      */
     public static List<Account> getFirstTenAccounts() {
-        return null; // Replace null with the variable you used to store the result
-    }
+        // Query for the first 10 Accounts ordered by Name
+        List<Account> firstTenAccts = [
+            SELECT Id, Name
+            FROM Account
+            ORDER BY Name
+            LIMIT 10
```

Comment:
One thing about vertical formatting you'll want to consider once you select more and more fields is also breaking down your fields you select. This is preference, but I wrap mine if they are > 3 fields I'm selecting usually.
```suggestion
            SELECT 
                    Id,
                    Name
            FROM Account
            ORDER BY Name
            LIMIT 10
```